docs/mp/lra/01_introduction.adoc

```diff
@@ -0,0 +1,115 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+= About Long Running Activities (LRA) in Helidon MP
+:h1Prefix: MP
+:pagename: intro-lra-in-mp
+:description: Introduction to LRA in Helidon MP
+:keywords: helidon, java, lra, mp, microprofile
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.microprofile.cors/io/helidon/microprofile/lra
+:helidon-tag: https://github.com/oracle/helidon/tree/{helidon-version}
+:quickstart-example: {helidon-tag}/examples/quickstarts/helidon-quickstart-mp
+:lra-spec: https://github.com/eclipse/microprofile-lra/blob/master/spec/src/main/asciidoc/microprofile-lra-spec.adoc
+:helidon-mp-lra-example: {helidon-tag}/examples/microprofile/lra
+:common-page-prefix-inc: ../../shared/lra/common_shared.adoc
+:mp-pages-ref-prefix: mp/lra
+:mp-using-lra-ref: {mp-pages-ref-prefix}/02_using-lra.adoc
+:mp-lra-config-ref: {mp-pages-ref-prefix}/03_configuration-with-lra-mp.adoc
+:helidon-variant: MP
+
+Microservices create challenges for data consistency and integrity that necessitate changes in the transaction processing and data patterns used by them. Helidon MP now has its own LRA implementation that adds additional features to the existing MicroProfile LRA specification.
+
+
+== Overview
+As stated in the MicroProfile 4.0 specification, link:{lra-spec}[MicroProfile LRA], "[...] introduces an API for loosely-coupled services to coordinate long running activities in such a way as to guarantee a globally consistent outcome without the need to take locks on data.” 
+
+Helidon MP supports the features described in the Microprofile spec, but also includes additional features: 
+
+
+* Support for messaging with AQ and Kafka
+* DB persistence of tx/LRA records and config for messaging
+* Future JSON support (if desired). Currently Helidon supports relational/SQL, but in the future we may provide flatfile or other persistence support.
+* Support for high availability (HA) and Kubernetes 
+
+== Messaging Support with AQ and Kafka
+
+The support for MicroProfile Messaging specification mimics the behavior of JAX-RS path as closely as possible. 
+For example, any aspect that applies to LRA-related JAX-RS headers applies to messaging properties as well.
```

Comment:
Do we know what a "LRA-related JAX-RS header" might be yet?

Reply:
This relates to the "Despite being in the "ws.ra" package, the `org.eclipse.microprofile.lra.annotation.ws.rs.LRA.LRA_HTTP_CONTEXT_HEADER` value
 will be reused as the key for AQ message properties, and Kafka K/V pairs will be used for LRA functionality."  part above.   
 
 We could state that here instead and replace "Despite being in the "ws.ra" package" with "For example,"